bubble_sort.rb

```diff
@@ -0,0 +1,32 @@
+def bubble_sort(argument)
+  loop do
+    changed = false
+    for i in 0...array.size - 1
```

Comment:
* [ ] This method is failing to work because ```array``` is not defined, kindly correct this
```
 argument=[2,1,3,4,0]
p bubble_sort(argument)
```

Reply:
Thanks for the comment! Updated.